src/components/icons.tsx

```diff
@@ -200,4 +200,52 @@ export const Icons = {
       <path fill="#ffba08" d="M12 12h10v10H12z" />
     </svg>
   ),
+  trash: (props: IconProps) => (
```

Comment:
This is not a maintainable approach to Icons, please use either FontAwesome or any other font based SVG icon provider.

Reply:
Use Phosphor library for general icons, keep only vendors icons